packages/riipen-ui/src/components/Menu.test.jsx

```diff
@@ -0,0 +1,333 @@
+import { mount } from "enzyme";
+import toJson from "enzyme-to-json";
+import React from "react";
+import ReactTestUtils from "react-dom/test-utils";
+
+import Menu from "./Menu";
+import MenuItem from "./MenuItem";
+import AppBar from "./AppBar";
+
+describe("<Menu>", () => {
+  it("renders without errors", () => {
+    let error;
+
+    try {
+      mount(<Menu />);
+    } catch (e) {
+      error = e;
+    }
+
+    expect(error).toEqual(undefined);
+  });
+
+  it("renders correct snapshot", () => {
+    const wrapper = mount(<Menu />);
+
+    expect(toJson(wrapper)).toMatchSnapshot();
+  });
+
+  describe("default props", () => {
+    it("sets correct default props", () => {
+      const defaultProps = new Menu().type.defaultProps;
+
+      const wrapper = mount(<Menu />);
+
+      expect(wrapper.find("Menu").props().color).toEqual(defaultProps.color);
+      expect(wrapper.find("Menu").props().closeOnClick).toEqual(
+        defaultProps.closeOnClick
+      );
+      expect(wrapper.find("Menu").props().isOpen).toEqual(defaultProps.isOpen);
+      expect(wrapper.find("Menu").props().keepOnScreen).toEqual(
+        defaultProps.keepOnScreen
+      );
+      expect(wrapper.find("Menu").props().popoverStyles).toEqual(
+        defaultProps.popoverStyles
+      );
+      expect(wrapper.find("Menu").props().variant).toEqual(
+        defaultProps.variant
+      );
+    });
+  });
+
+  describe("anchorEl prop", () => {
+    it("renders anchor element that menu is attached to", () => {
+      const anchorEl = ReactTestUtils.renderIntoDocument(<AppBar />);
+
+      const wrapper = mount(<Menu anchorEl={anchorEl} />);
+
+      expect(wrapper.find("Popover").props().anchorEl).toEqual(anchorEl);
+    });
+  });
+
+  describe("anchorPosition prop", () => {
+    it("passes anchorPosition prop to correct child", () => {
+      const anchorPosition = { horizontal: "left", vertical: "bottom" };
+
+      const wrapper = mount(<Menu anchorPosition={anchorPosition} />);
+
+      expect(wrapper.find("Popover").props().anchorPosition).toEqual(
+        anchorPosition
+      );
+    });
+  });
+
+  describe("autoFocus prop", () => {
+    it("passes autoFocus prop to correct child", () => {
+      const autoFocus = true;
+      const anchorEl = ReactTestUtils.renderIntoDocument(
+        <div>
+          <AppBar />
+        </div>
+      );
+
+      const wrapper = mount(
+        <Menu anchorEl={anchorEl} autoFocus={autoFocus}>
+          <MenuItem />
+        </Menu>
+      );
+
+      expect(wrapper.find("MenuList").props().autoFocus).toEqual(autoFocus);
+    });
+  });
+
+  describe("children prop", () => {
+    it("renders given children", () => {
+      const child = <div>child</div>;
+      const anchorEl = ReactTestUtils.renderIntoDocument(
+        <div>
+          <AppBar />
+        </div>
```

Comment:
Do you need the wrapping `<div>`'s here? Seems like in some places you use them and others you don't.

Reply:
The wrapping divs allow for Popover's children to be rendered. So I think they are needed when testing props passed to MenuList. 

The wrapping divs had to be left out for the anchorEl test at line 53, otherwise I would also have to pass in the required children for MenuList.

Reply:
Ok cool, that's good to know!